.gitignore

```diff
@@ -68,3 +68,4 @@ distTmp/
 outTmp/
 /test.output
 /kotlin-native/dist
+kotlin-ide/
```

Comment:
Looks like this has been merged by my other PR. We are following https://docs.google.com/document/d/1Nw-rVfdgwD-7x2XjDp8hskEkFV9aBtjYhiJy1PmxGyc/edit#heading=h.a1wqyqvjwx72 to set up our build and `kotlin-ide` is where the `intellij-kotlin` repo is placed. Soon we won't need it since we are migration to kotlin+intellij setup and no longer need intellij-kotlin repo.